csi/controller.go

```diff
@@ -0,0 +1,230 @@
+/*
+Package csi is CSI driver interface for OSD
+Copyright 2017 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package csi
+
+import (
+	"fmt"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"go.pedge.io/dlog"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// ControllerGetCapabilities is a CSI API functions which returns to the caller
+// the capabilities of the OSD CSI driver.
+func (s *OsdCsiServer) ControllerGetCapabilities(
+	ctx context.Context,
+	req *csi.ControllerGetCapabilitiesRequest) (*csi.ControllerGetCapabilitiesResponse, error) {
```

Comment:
Can you put all the function prototypes in the following format
func (s *osd) funcName(
arg1 string,
arg2 string,
) (return values)

We try to follow this [styleguide](https://github.com/libopenstorage/openstorage/blob/master/STYLEGUIDE.md)

Reply:
Ah! I forgot the last return. Yeah, i'll make these changes.